index.md

```diff
@@ -1,49 +1,41 @@
 ---
-layout: default
+layout: home
 title: Home
 section: Overview
 ---
+<p class='alert-big'>Origami is a group of services, modules and tools to make building websites at the FT simpler and faster.</p>
 
-<p class="alert-big">Origami is a set of standards, conventions and tools for building websites at the FT.</p>
+<h2> <i class="fa fa-cubes"></i> Origami modules and services</h2>
 
-<p class="alert-big">Origami aspires to create a <strong>unified style and experience for FT websites</strong>, and <strong>make web development at the FT faster</strong>.</p>
+At the heart of Origami are its services and modules.
 
-This standard is equally for component developers and those who want to build websites using existing components.  These two roles will be distinguished in this standard as **component developer** and **product developer**.
+Origami's modules are bits of HTML, CSS and JavaScript to include in your page. Some of them, like [o-gallery](http://registry.origami.ft.com/components/o-gallery), are fully formed ready to use components. Some of them, like [o-colors](http://registry.origami.ft.com/components/o-colors) or [o-tracking](http://registry.origami.ft.com/components/o-tracking) are tools to incorporate into your own pages and build with.
 
-The things being built by component developers are **components**, while the sites, products, tools or services being build by the product developers as new services to FT readers are **product applications**.
+Regardless of what they're made of, these modules have been comprehensively tested, comply with web accessibility standards, work or degrade gracefully in all of the browsers we support, and conform to the FT brand guide. To see a complete list of the modules visit [registry.ft.com](http://registry.ft.com).
```

Comment:
I didn't mean anything so concrete as either of those things, but rather some wording that means components are visually 'FT like'. If you'd prefer it could be re-written as "on brand" which is the wording used elsewhere on origami.ft.com. Assuming "on brand" is OK, does "that fit with the FT's brand" work?